src/Illuminate/Foundation/Console/OptimizeCommand.php

```diff
@@ -96,19 +100,22 @@ protected function compileClasses()
     }
 
     /**
-     * Get the node traverser used by the command.
+     * Get the class preloader used by the command.
      *
-     * @return \ClassPreloader\Parser\NodeTraverser
+     * @return \ClassPreloader\ClassPreloader
      */
-    protected function getTraverser()
```

Comment:
It would be good if we can avoid removing `protected` method on a patch release since someone (like me) who build packages that replace this command can have some time to plan upgrade.